server/api/index.js

```diff
@@ -6,13 +6,42 @@
  * it can be accessed via /:resource
  */
 import {Router as router} from 'express';
+import {BigQuery} from '@google-cloud/bigquery';
```

Comment:
nit: Alphabetize according to https://google.github.io/styleguide/jsguide.html

There is no official way to alphabetize this module import syntax, but what I've been doing is this:

1.- First alphabetize default imports
2.- Then, alphabetize the destructured exports by inside, (e.g) from `import { Foo, Bar } from 'path'` to `import { Bar, Foo } from 'path';
3.- After the default imports, alphabetize the destructured ones by the first element of inside `{}`

For this case would be:

```
import mixedApi from './mixed-content';
import {BigQuery} from '@google-cloud/bigquery';
import {Router as router} from 'express';
```

Same for other imports

Reply:
Like this?:
```
import MixedContentPercentageHistogram
  from './mixed-content-percentage-histogram';
import React from 'react';
import TopCountriesWithMoreWebsitesWithMixedContent
  from './top-countries-with-more-websites-with-mixed-content';
import TopGovernmentWebsitesWithMixedContent
  from './top-government-websites-with-mixed-content';
import TopWebsitesWithMixedContent
  from './top-websites-with-mixed-content';
```
It seems strange to see react in the middle

Reply:
Looks good.

I know, usually React is at the top using some other standards, but Google is not specific about that 😢